netutils/config/parser.py

```diff
@@ -928,7 +929,7 @@ def _build_nested_config(self, line):
             IndexError: When the number of parents does not match the expected deindent level.
         """
         if "[" in line:
-            line = self.uncommon_data.get(line.split('"')[1])
+            line = self.uncommon_data.get(line.split('"')[1])  # type: ignore
```

Comment:
Should perform a bit of explicit optional handling here to get rid of the ignore comment.